sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -200,6 +200,20 @@ public partial class RequestContext
         public void AddPolicy(Azure.Core.Pipeline.HttpPipelinePolicy policy, Azure.Core.HttpPipelinePosition position) { }
         public static implicit operator Azure.RequestContext (Azure.ErrorOptions options) { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public partial struct RequestFailedDetails
+    {
+        private object _dummy;
+        private int _dummyPrimitive;
+        public RequestFailedDetails(Azure.ResponseError? error, System.Collections.Generic.IDictionary<string, string>? additionalInfo) { throw null; }
```

Comment:
Curious: why are both inputs nullable? Is there a case where we'd have error details without an error?

Reply:
Does it make sense to make an allocation for a type that contains only null properties?  Could you return null instead?

Reply:
It's in the exception path, so I don't think that matters all too much. I tried a different shape in the latest iterations.